### PR TITLE
Add option to balance lockpick ring.

### PR DESCRIPTION
**Pull request by robbintt**
Default to off since this is a waste of time unless you're a thief or frugal.

### CODE REVIEW

File: pick.lic [hunk 1]
```diff
@@ -231,6 +232,11 @@ class Pick
     @equipment_manager.wear_items(@removed_items)
   end
 
+  def balance_lockpick_container
+    return unless @use_lockpick_ring and @balance_lockpick_container
```
Comment (wstampley) [inline pick.lic:236]: I think you'll need something at the top to pull @settings.balance_lockpick_container into the class variable @balance_lockpick_container, and probably to add it to the debug output of all the settings.

File: pick.lic [hunk 2]
```diff
@@ -231,6 +232,11 @@ class Pick
...
+    bput("turn my #{@settings.lockpick_container} to best", "You fiddle with", "You think about it")
```
Comment (wstampley) [inline pick.lic:237]: This version of pick doesn't do the Ruby include of any commons to make it easier to unit test, so you will need to prefix this bput with DRC.

File: profiles/base.yaml [hunk 3]
```diff
@@ -999,6 +999,7 @@ skeleton_key: skeleton key
 # Set to false to use lockpicks instead.
 use_skeleton_key: false
 use_lockpick_ring: true
+balance_lockpick_container: false
```
Comment (wstampley) [inline profiles/base.yaml:1002]: Other new properties that are only used the new pick were nested under "pick:", i.e.
```
pick:
  balance_lockpick_container: false
```
Obviously not a 'must change', just something to consider.

File: pick.lic [hunk 4]
```diff
@@ -39,6 +39,7 @@ class Pick
 
     @use_lockpick_ring = @settings.use_lockpick_ring
     @lockpick_container = @settings.lockpick_container
+    @balance_lockpick_container = @settings.pick['balance_lockpick_container']
```
Comment (rpherbig) [inline pick.lic:42]: Should this be output inside the block on line 101?

**Comment by rpherbig** [comment]
Code changes look good aside from the two comments ✔️

**Pull request merged**